src/dom/script_request.js

```diff
@@ -13,6 +13,8 @@
   }
   var prototype = ScriptRequest.prototype;
 
+  var _document = Pusher.runtime.getDocument();
```

Comment:
Doing a request for a file via dynamically creating a `script` tag feels browser runtime specific. So maybe this would be moved into `Pusher.Runtime` if we proceed?